x-pack/solutions/observability/plugins/slo/server/services/find_slo_definitions.ts

```diff
@@ -23,7 +23,8 @@ export class FindSLODefinitions {
 
   public async execute(params: FindSLODefinitionsParams): Promise<FindSLODefinitionsResponse> {
     const result = await this.repository.search(params.search ?? '', toPagination(params), {
-      includeOutdatedOnly: params.includeOutdatedOnly === true ? true : false,
+      includeOutdatedOnly: params.includeOutdatedOnly === true,
+      tags: params.tags,
```

Comment:
Transform the data coming from the router (string or undefined) to something that makes more sense for the repository. `string` is too vague, `string[]` would be easier to understand.

```suggestion
      tags: params.tags?.split(",") ?? []
```